KlockanAPI/src/KlockanAPI.Infrastructure/Data/KlockanContext.cs

```diff
@@ -4,7 +4,9 @@
 
 namespace KlockanAPI.Infrastructure.Data;
 
-public class KlockanContext : DbContext
+public class 
+    
+    KlockanContext : DbContext
```

Comment:
check the format of the line, it should be in one line
public class KlockanContext : DbContext